### PR TITLE
pull in openshift login plugin for login via openshift oauth

### PR DESCRIPTION
**Pull request by gabemontero**
Waiting on the openshift-login rpm to be built and puddles updated before merging.

But README updates are available for commentary, and I'll at least test out centos in the interim.

@bparees fyi

### CODE REVIEW

**Comment by gabemontero** [comment]
the openshift-login rpm is available now ... was able to build rhel image in my personal ami

Trying now on ci.openshift (rewording commit and pushing triggered PR testing)

**Comment by gabemontero** [comment]
OK, PR test has passed.  Going to merge to get the image updated with new plugins / new plugin versions.  For the README, we can tweak separately.  Also, the openshift-docs review outcome for oauth can serve as a reminder and potential input driver for more README changes.

File: README.md [hunk 1]
```diff
@@ -165,9 +171,34 @@ $ docker run -d -e JENKINS_PASSWORD=<password> -v /tmp/jenkins:/var/lib/jenkins
...
+* hudson.model.Item.Build
+* hudson.model.Item.Configure
+* hudson.model.Item.Create
+* hudson.model.Item.Delete
+* hudson.model.Item.Workspace
+* hudson.scm.SCM.Tag
+
+Permissions for users can be changed after they are initially established.  Note, changes to permissions in Jenkins do not result in changed permissions for the users in {product-title}.
+
+Also note, the OpenShift Login plugin polls the OpenShift API server for permissions and will update the permissions for each user currently defined in Jenkins.  You can control how often the polling
```
Comment (bparees) [inline README.md]: if the user permissions are edited in jenkins, will the changes get reverted if the permissions also change in openshift?  how do conflicts get resolved?

**Comment by bparees** [comment]
I think we just need to make the behavior clear in the docs.

Ben Parees | OpenShift

**Comment by gabemontero** [comment]
OK I'll work on making it clearer.

**Pull request merged**